shared/Features/FeatureFlags.cs

```diff
@@ -8,13 +8,17 @@ namespace GovUk.Education.SearchAndCompare.UI.Shared.Features
     public class FeatureFlags : IFeatureFlags
     {
         private readonly IConfiguration _config;
-        private const string APPLY_2019 = "FEATURE_APPLY_2019";
+        private const string FeaturePrefix = "FEATURE_";
+        private const string Apply2019Key = FeaturePrefix + "APPLY_2019";
+        private const string MapsKey = FeaturePrefix + "MAPS";
 
         public FeatureFlags(IConfiguration config)
         {
             _config = config;
         }
-        public bool Apply2019 => ShouldShow(APPLY_2019);
+
+        public bool Apply2019 => ShouldShow(Apply2019Key);
```

Comment:
This can go now 😆

Reply:
we can remove this in a subsequent PR

Reply:
https://trello.com/c/4XxSadm2/475-remove-apply-feature-flag